docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,404 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it boots from a volume
+(which we generally recommend since it affords more flexibility than
+booting from an image). You may still have boot-from-image servers,
+though. To ensure a particular server is of that type, you can go to the
```

Comment:
I use "ensure" as a verb meaning "ascertain that something is in a *desired* state". That is not the case here, so I would instead suggest:

```suggestion
though. To verify that a particular server is of that type, you can go to the
```